templates/writing-task-topics.rst

```diff
@@ -0,0 +1,240 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+###################
+Writing Task Topics
+###################
+
+This page describes how to document science pipelines tasks for the
+`pipelines.lsst.io <http://pipelines.lsst.io>`_ documentation project.
+
+Using the template
+==================
+
+We provide a standardized template for all task documentation
+pages. This required template ensures consistency and style, and also
+provides you tools for automatically extracting documentation from
+your code.
+
+You can obtain the template from :doc:`here <template-for-tasks>`.
+
+Installing the template
+-----------------------
+
+To install the template, copy it into the `doc/tasks` directory of
+your stack package. Then name the file after the task class, with a
+`.rst` extension. For example: `doc/tasks/ProcessCcdTask.rst`.
+
+Filling out the template
+------------------------
+
+The template includes brief instructions as reStructuredText
+comments. Each component of the documentation page is described in
+more detail on this page, below,.
+
+
+
+.. _task-topics-intro:
+
+Introduction
```

Comment:
Ok... this seems a more substantial change, expanding a number of the subsections in the intro, but i realize that it was getting kind of top-heavy so that it makes sense.

Seems like then there would be no more "Introduction" per se, since it's all in these Intro Subsections?

I implemented this as i understood it, let's see your thoughts.